modules/ROOT/pages/promote-app-prod-env-design-center.adoc

```diff
@@ -9,29 +9,29 @@ endif::[]
 
 All Mule applications that you create with Flow Designer are created in the Design environment. When a Mule application that you've created is ready to go live, you can deploy it to another environment, if your organization allows you to do so. The target environment must have enough vCores to accommodate the application; otherwise, deployment will fail.
 
-As you deploy your application, you can change connector configurations and reconnection strategies. You need to do this if there are differences between the Design and target environments that would necessitate such changes.
+As you deploy your application, you can change connector configurations and connections. You need to do this if there are differences between the testing and target environments that would necessitate such changes. For example, if you tested your app in the *Sandbox* environment and now want to deploy it to a production environment, you most likely need to replace the connections that each connector used for connecting to test systems with connections for connecting to production systems.
 
-== Procedure
-. Click *Deploy* in the top-right corner of Flow Designer.
+After you deploy your application, the values for all of its properties are secured in Runtime Manager in the *Properties* section of the *Settings* page for the application. To see the list of an application's properties in Runtime Manager, click *Applications*, select the application, click *Settings*, and click *Properties*.
+
+== To Deploy a Mule App from Flow Designer
+. Click the arrow in the top-right corner of Flow Designer and then select *Deploy application*.
 +
-image::deploy-dialog.png[]
+.The *Deploy application* option appears in this menu.
+image::fd-deploy-option.png[The location of the Deploy application option]
+<1> Click the arrow.
+<2> Select the option.
 
 . Select the CloudHub environment to deploy the application to.
```

Comment:
I think Cloudhub is a name that is not meant to be used in the UI. I believe the UI name of the product is Runtime Manager.

Reply:
However, the way all of the MuleSoft documentation describes it, environments are in CloudHub. The Runtime Manager documentation talks about managing things that are in CloudHub environments.